CONTRIBUTING.md

```diff
@@ -185,16 +185,30 @@ Here is the svg for the Adobe Photoshop icon as an example:
 
 Icon metadata should be added to the `_data/simple-icons.json` file. Each icon in the array has three required values:
 
-  * The `title` of the new SVG.
-  * A `hex` color value that matches the brand's primary color. All uppercase and without the `#` pound symbol.)
-  * The `source` URL of the logo being used. There are [more details below](#source-guidelines).
+* The `title` of the new SVG.
+* A `hex` color value that matches the brand's primary color. All uppercase and without the `#` symbol.
+* The `source` URL of the logo being used. There are [more details below](#source-guidelines).
 
-Additionally, there are also optional fields that may provided for an icon:
+Here is the object of a fictional brand as an example:
 
-- The `guidelines` may be used to specify the URL of the brand's guidelines/presskit/etc. This is useful if the SVG file was sourced from a different place.
-- The `license` may be used to record the license under which the icon is available. This is an object with a `type` and `url`. The `type` should be an [SPDX License ID](https://spdx.org/licenses/) or `"custom"`, the `url` is optional unless the `type` is `"custom"`.
+```json
+{
+    "title": "A Fictional Brand",
+    "hex": "123456",
+    "source": "https://www.a-fictional-brand.org/logo"
+}
+```
 
-Here is the object of a fictional brand as an example:
+Make sure the icon is added in alphabetical order. If you're in doubt, you can always run `npm run our-lint` - this will tell you if any of the JSON data is in the wrong order.
```

Comment:
Perhaps adding here a link pointing to [Testing Package Locally](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#testing-package-locally) section?

 ```suggestion
Make sure the icon is added in alphabetical order. If you're in doubt, you can always run `npm run our-lint` - this will tell you if any of the JSON data is in the wrong order (refers to [Testing Package Locally](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#testing-package-locally) if you don't know how to setup the environment locally).
```

Reply:
I'm not sure if we should link there every time we mention NPM scripts - if we decide to do that we should also add it throughout [section 3](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#3-optimize-the-icon) and link to the docker section every time docker is mentioned. I think this relates a bit to https://github.com/simple-icons/simple-icons/issues/5018 and to what extend we should go to help beginners. That said, I think it's still a good suggestion! But I think the best we can reasonable do in this regard is link to the "Testing Package Locally" section in the [Table of Contents](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#table-of-contents). (And perhaps improving the name of the section.) But I think we should leave that for a separate PR.

Reply:
Okay, lets leave this in a separate PR